pydoctor/templatewriter/summary.py

```diff
@@ -260,17 +242,11 @@ def loader(self):
     def __init__(self, system):
         self.system = system
 
+    # Deprecated: pydoctor's templates no longer use this, but it is kept
+    #             for now to not break customized templates like Twisted's.
     @renderer
     def project_link(self, request, tag):
-        if self.system.options.projecturl:
-            return tags.a(href=self.system.options.projecturl)(
-                self.system.projectname)
-        else:
-            return self.system.projectname
-
-    @renderer
-    def project(self, request, tag):
-        return self.system.projectname
+        return self.project_tag
```

Comment:
I would say that project_link should also be moved to util.Page base page as it should be common.


Now I see that the template no longer uses it...but I don't know if that is the right thing to do :)